microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -645,6 +645,18 @@ By default, while browsing **InPrivate**, Microsoft does not collect any informa
 Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.  You can stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing. Open **Start** > **Settings** > **Privacy** and turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
 
 
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When working with the work profile, the organization can see the following data:
```

Comment:
need more verbose lead-in sentence to establish context

Reply:
New phrasing: When working with the work profile, the organization **may** see the following data:

Reply:
i'll revise

Reply:
changed to 'may'